src/serial_port.c

```diff
@@ -17,10 +17,7 @@ size_t serial_port_printf(com_port port, char *string)
 {
   size_t length = strlen(string);
 
-  for (size_t i = 0; i < length; i++)
-  {
-    serial_port_putchar(port, string[i]);
-  }
+  for (size_t i = 0; i < length; i++) serial_port_putchar(port, string[i]);
```

Comment:
Please do not change the code style. For loops should always have braces because that is a lot more readable.